### PR TITLE
[RIS-309] Implement Path Based Product Routing (Static)

### PR DESCRIPTION
**Pull request by math-marcellino**
## Done
1. Static routing for `binance/markets` and `arbitrum/markets`
2. Display style based on the network
3. Handle change market when switch network
4. Show `switch network` button when connecting to wrong network (e.g. user is in bsc market but their metamask is connected to arbitrum, then the `switch network` button shown)

## TODO
1. Implement token pages (ETHRISE, ETHBRISE, BTCBRISE, CAKERISE, BNBRISE)
2. Implement new behavior on portfolio page

### CODE REVIEW

File: modules/marketsPage/components/BackgroundGradient.tsx [hunk 1]
```diff
@@ -27,13 +29,13 @@ const BackgroundGradient: FunctionComponent<BackgroundGradientProps> = ({}) => {
                     </g>
                 </svg>
             </div>
-            {chain.chain.id === customChains.bsc.id && (
+            {chainID === customChains.bsc.id && (
```
Comment (asaadam) [inline modules/marketsPage/components/BackgroundGradient.tsx]: better to rewrite to new component

and using if else or switch case. 

it's more readable

**Review by asaadam — CHANGES_REQUESTED** [review]

File: modules/marketsPage/components/BackgroundGradient.tsx (hunk 1 above)
Comment (math-marcellino) [reply modules/marketsPage/components/BackgroundGradient.tsx]: fixed sir, please kindly check again

**Review by asaadam — APPROVED** [review]
Approved !

wait for @pyk

File: components/v1/Buttons/ConnectWalletDesktop.tsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import Link from "next/link";
+import { useRouter } from "next/router";
```
Comment (pyk) [inline components/v1/Buttons/ConnectWalletDesktop.tsx:2]: @math-marcellino why we need to use `next/router` ?

**Comment by pyk** [comment]
@math-marcellino looks good, on the homepage we may need do change "launch app" to something that show we are available on multichain

maybe multiple launch buttons? not sure

File: components/v1/Buttons/ConnectWalletDesktop.tsx (hunk 2 above)
Comment (math-marcellino) [reply components/v1/Buttons/ConnectWalletDesktop.tsx:2]: We use it detect the `pathname` sir. So if the `pathname` contain `binance`, then the `default chain` for that page will be BSC. Implementation can be seen in code below:
```
const selectedChain = router.pathname.includes("binance") ? customChains.bsc : Chains.arbitrumOne;
```

Comment (pyk) [reply components/v1/Buttons/ConnectWalletDesktop.tsx:2]: @math-marcellino gotcha.

The problem is that if we have more chains, this line will be too long

```
const selectedChain = router.pathname.includes("binance") ? customChains.bsc : Chains.arbitrumOne;
```

but it's nice problem to have! let's just use this for now and revisit it later

**Review by pyk — APPROVED** [review]

**Pull request merged**